ocs_ci/ocs/cluster.py

```diff
@@ -1712,6 +1713,46 @@ def check_ceph_health_after_add_capacity(
     ), "Data re-balance failed to complete"
 
 
+def validate_existence_of_blocking_pdb():
+    """
+    Validate creation of PDBs for OSDs.
+    1. Versions lesser than ocs-operator.v4.6.2 have PDBs for each osds
+    2. Versions greater than or equal to ocs-operator.v4.6.2-233.ci have
+    PDBs collectively for osds like rook-ceph-osd
+
+    Returns:
+        bool: True if blocking PDBs are present, false otherwise
+
+    """
+    pdb_obj = ocp.OCP(
+        kind=constants.POD_DISRUPTION_BUDGET, namespace=defaults.ROOK_CLUSTER_NAMESPACE
+    )
+    pdb_obj_get = pdb_obj.get()
+    osd_pdb = []
+    for pdb in pdb_obj_get.get("items"):
+        if (
+            constants.MDS_PDB not in pdb["metadata"]["name"]
+            and constants.MON_PDB not in pdb["metadata"]["name"]
+        ):
+            osd_pdb.append(pdb["metadata"]["name"])
```

Comment:
Since you're looping over the list again later (in the "else" block), you might prefer to append the object itself for later use